src/pages/PageNotFound/PageNotFound.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+const PageNotFound = () => (
```

Comment:
⛏️ Could you rename all of these components to `NotFound` instead of `PageNotFound`?

Reply:
Addressed: d612efc22a699d66c1ab524fa76d86e4b6c65096